Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -976,7 +976,10 @@ def imread(
         # Increase dimension if last dimension is not of size one.
         if increase_dimension and image_IO.GetDimensions(dimension - 1) != 1:
             dimension += 1
-        ImageType = itk.Image[pixel_type, dimension]
+        if itk.template(pixel_type)[0] is itk.VariableLengthVector:
```

Comment:
-> `if isinstance(pixel_type, itk.support.template_class.itkTemplate) and itk.template(pixel_type)[0] is itk.VariableLengthVector`

Reply:
```text
  File "C:\Dev\ITK-py\Wrapping\Generators\Python\itk\support\extras.py", line 983, in imread
    ImageType = itk.Image[pixel_type, dimension]
  File "C:\Dev\ITK-py\Wrapping\Generators\Python\itk\support\template_class.py", line 529, in __getitem__
    raise itk.TemplateTypeError(self, key)
itk.support.extras.TemplateTypeError: itk.Image is not wrapped for input type `itk.VariableLengthVector[itk.UC], int`.
```
Indeed, `itk.Image` is not wrapped for `itk.VariableLengthVector`. Should we add wrapping of VLV for all scalar types? Should we make it configurable via a CMake option?

Reply:
> ImageType = itk.Image[pixel_type, dimension]

We should never hit this line if `pixel_type` is VLV. VLV corresponds to the pixel type for `itk.VectorImage`. We cannot / should not wrap `itk.Image` for it.